LearnosityDemo/Pages/QuestionsAPIDemo.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@page
+@model LearnosityDemo.Pages.QuestionsAPIDemoModel
+@{
+    ViewData["Title"] = "Learnosity Example: Standalone Assessment";
+    ViewData["TopJS"] = "<script src=\"https://questions.learnosity.com/?verison=developer\"></script>";
```

Comment:
typo "verison".
is "developer" correct?

Reply:
@bhavya-shukla-lrn @markus-liang-lrn Good pickup Markus, yes it should be "version", and let's use "latest-lts" instead of "developer".

Reply:
@TheRealEdDawson I think the change for latest-lts will apply in all sdks?

Reply:
@bhavya-shukla-lrn Yes, that's right.